### PR TITLE
introduced a run time config of the number type from the html control

### PR DESCRIPTION
**Pull request by nirsmadar**

### CODE REVIEW

**Comment by bighappyface** [comment]
@nirsmadar thanks for the help! Some highl-level feedback:

1. Please squash this down to one commit
2. Please use the conventional changelog commit message format

Please read [CONTRIBUTING.md](https://github.com/hodgepodgers/ng-intl-tel-input/blob/d886de9959d81724578fb9251dbbc4964fa3e281/README.md) for more guidelines.

File: index.html [hunk 1]
```diff
@@ -82,6 +82,57 @@
...
+      <div class="well">
+        <code>&lt;input type=&quot;text&quot; name=&quot;tel&quot; ng-model=&quot;tel&quot; ng-intl-tel-input data-default-number-type=&quot;FIXED_LINE&quot;&gt;</code>
+      </div>
+
+      <div class="row">
+        <div class="col-md-6">
+          <form name="form2">
+            <div class="form-group" ng-class="{'has-error': form2.tel.$invalid && form2.tel.$touched, 'has-success': form2.tel.$valid}">
+              <label for="tel">Telephone number</label>
+              <input type="text" class="form-control" id="tel-dnt-f" name="tel" ng-model="tel4" ng-intl-tel-input data-default-country="IL" data-default-number-type='FIXED_LINE'>
```
Comment (bighappyface) [inline index.html]: Inconsistent quotes

File: index.html [hunk 2]
```diff
@@ -82,6 +82,57 @@
...
+      <div class="well">
+        <code>&lt;input type=&quot;text&quot; name=&quot;tel&quot; ng-model=&quot;tel&quot; ng-intl-tel-input data-default-number-type=&quot;MOBILE&quot;&gt;</code>
+      </div>
+
+      <div class="row">
+        <div class="col-md-6">
+          <form name="form2">
+            <div class="form-group" ng-class="{'has-error': form2.tel.$invalid && form2.tel.$touched, 'has-success': form2.tel.$valid}">
+              <label for="tel">Telephone number</label>
+              <input type="text" class="form-control" id="tel-dnt-m" name="tel" ng-model="tel3" ng-intl-tel-input data-default-country="IL" data-default-number-type='MOBILE'>
```
Comment (bighappyface) [inline index.html]: Inconsistent quotes

File: ng-intl-tel-input.directive.js [hunk 3]
```diff
@@ -14,6 +14,10 @@ angular.module('ngIntlTelInput')
           if (attr.defaultCountry) {
             ngIntlTelInput.set({defaultCountry: attr.defaultCountry});
           }
+          // Override default number type.
+          if (attr.defaultNumberType) {
```
Comment (bighappyface) [inline ng-intl-tel-input.directive.js]: I feel like this should check that the proper `FIXED_LINE` or `MOBILE` values are being assigned, but the underlying library may help cover that base.

**Comment by bighappyface** [comment]
@SyntaxStacks check this out

**Comment by nirsmadar** [comment]
hi thanks :) sorry for all the mess, as this is my first ever pull request.. i'll try to do better tomorrow.

**Comment by bighappyface** [comment]
@nirsmadar no worries. We will get you through.

**Comment by bighappyface** [comment]
@nirsmadar any progress?

**Comment by nirsmadar** [comment]
CI tests passed as far as I can see.
Again- really sorry for the messy pull req- I've never used any of those systems (protractor, karma etc.) and I'm learning a lot here.

Hope the code is good enough though.

**Comment by SyntaxStacks** [comment]
`git rebase -i HEAD~6`

That command will allow you to squash your commits into one. If this is your first time doing this kinds thing, I suggest that you create another branch and play around there instead of the branch for this PR. Once you have the hang of it please, push rebase this branch and push it back up. 

Great work on the test. Glad you're learning and let me know if I can help. :]

**Comment by nirsmadar** [comment]
tried to do it- hope i didn't make it worse :)

**Comment by nirsmadar** [comment]
Hey guys- any progress? something i need to do?

**Comment by SyntaxStacks** [comment]
It appears that you didnt force push the squashed commits and merged you changes you had upstream. 

Please squash this down to one commit

**Comment by SyntaxStacks** [comment]
Please see here for info on squashing
https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#Changing-Multiple-Commit-Messages

**Comment by bighappyface** [comment]
@nirsmadar any movement on this?

**Comment by nirsmadar** [comment]
i ran into problems with the squash and abandoned it. I'm afraid i did a mess there..

**Pull request closed without merging**